### PR TITLE
fix: missing error handling

### PR DESCRIPTION
**Pull request by Jakub-Vacek**
This PR fixes missing error handling when getting profile ast.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have updated the documentation accordingly. If you are changing **code related to user secrets** you need to really make sure that [security documentation](https://github.com/superfaceai/one-sdk-js/blob/main/SECURITY.md) is correct.
- [x] I have read the [CONTRIBUTING](https://github.com/superfaceai/one-sdk-js/blob/main/CONTRIBUTING.md) document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/core/errors/errors.helpers.ts [hunk 1]
```diff
@@ -507,6 +508,24 @@ export function unexpectedDigestValue(
   );
 }
 
+export function invalidResponseError(error: unknown): SDKExecutionError {
+  if (isRegistryErrorBody(error)) {
+    return new SDKBindError(
+      `Registry call failed with unexpected error: ${
+        error.detail ?? error.title
```
Comment (freaz) [inline src/core/errors/errors.helpers.ts:515]: suggestion(not-blocking): I would keep (show) both detail and title

File: src/core/registry/registry.test.ts [hunk 2]
```diff
@@ -647,6 +648,40 @@ describe('registry', () => {
...
+
+    it('throws error on invalid status code and response body with detail', async () => {
+      const mockBody = {
+        detail: 'Test',
+        title: 'Title',
+      };
+      const mockResponse = {
+        statusCode: 400,
+        body: mockBody,
+        headers: { test: 'test' },
```
Comment (freaz) [inline src/core/registry/registry.test.ts:660]: question: I don't know if relying only on body structure is right here. Server is returning header`Content-Type: application/problem+json` so the clients know what structure to expect.

**Review by freaz — APPROVED** [review]
Definitely missing part, only thing I am not sure about is skipping content type of the response. Otherwise looks ok.

File: src/core/registry/registry.test.ts (hunk 2 above)
Comment (Jakub-Vacek) [reply src/core/registry/registry.test.ts:660]: I can use `content-type` to decide.

**Pull request merged**